orc8r/cloud/go/services/configurator/test_init/test_service_init.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	testServiceMaxLoadSize = 10
+	TestServiceMaxLoadSize = 10
```

Comment:
Since we're renaming it: this is really more accurate to call it the "page" size
```suggestion
	TestServiceMaxPageSize = 10
```